bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyDiscoveryParticipant.java

```diff
@@ -166,6 +159,9 @@ public DiscoveryResult createResult(final ServiceInfo service) {
 
                     // create shellyunknown thing - will be changed during thing initialization with valid credentials
                     thingUID = ShellyThingCreator.getThingUID(name, model, mode, true);
+                }
+                if (e.isTimeout()) {
+                    logger.info("{}: {}", name, messages.get("discovery.failed", address, e.toString()));
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug.

Reply:
removed